### PR TITLE
docs: add image.preview.forceRender props

### PR DESCRIPTION
**Pull request by binyellow**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution
在使用Image.preview时
```js
setVisible(false);
setSrc(undefined);
```
发现preview新的图片时，`setVisible(true)`后在弱网情况下，img的src还是preSrc，看了下rc-util/PortalWrapper在visible为false时，children未reRender，[需要显示的设置`preview.forceRender = true`](https://github.com/react-component/util/blob/master/src/PortalWrapper.tsx#L233)

[network设置为slow3G，切换图片可以看到这个问题](https://ant.design/components/image-cn/#components-image-demo-preview-group)

所以在文档添加了这个参数

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     add image.preview.forceRender props      |
| 🇨🇳 Chinese |     添加Image.preview.forceRender参数      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/image/__tests__/index.test.tsx [hunk 1]
```diff
@@ -87,4 +87,30 @@ describe('Image', () => {
...
+        />
+      );
+    };
+
+    const { baseElement } = render(<PreviewImage />);
+    const maskDom = baseElement.querySelector('.ant-image-preview-wrap');
+    fireEvent.click(maskDom!);
+    expect(baseElement.querySelector('.ant-image-preview-img')?.getAttribute('src')).toBe(
+      null,
+    );
```
Comment (afc163) [inline components/image/__tests__/index.test.tsx:114]: 没太看懂这个用例，forceRender 是不是应该检查 img 在点击 preview 前已经提前渲染出来了。

Comment (afc163) [reply components/image/__tests__/index.test.tsx:114]: 以及 image 的 onload 会提前触发。

Comment (binyellow) [reply components/image/__tests__/index.test.tsx:114]: 1

File: components/image/__tests__/index.test.tsx [hunk 2]
```diff
@@ -87,4 +87,20 @@ describe('Image', () => {
...
+          visible: false,
+          src,
+          forceRender: true,
+        }}
+      />
+    );
+    const { baseElement } = render(<PreviewImage />);
+    expect(baseElement.querySelector('.ant-image-preview-root')).not.toBe(null);
+    baseElement.querySelector('.ant-image-preview-img')?.addEventListener('onLoad', onLoadCb);
+    expect(onLoadCb).not.toHaveBeenCalled();
```
Comment (afc163) [inline components/image/__tests__/index.test.tsx:104]: 应该是 toHaveBeenCalled ？

Comment (binyellow) [reply components/image/__tests__/index.test.tsx]: 好啦，辛苦大佬再看看

File: components/image/__tests__/index.test.tsx [hunk 3]
```diff
@@ -87,4 +87,21 @@ describe('Image', () => {
...
     expect(baseElement.querySelector('.container')?.children.length).not.toBe(0);
   });
+  it('Preview forceRender props', async () => {
+    const onLoadCb = jest.fn();
+    const PreviewImage: React.FC = () => (
+      <Image
+        preview={{
+          visible: false,
+          src,
+          forceRender: true,
```
Comment (afc163) [inline components/image/__tests__/index.test.tsx:97]: 去掉这句，这个用例还能过么？

Comment (binyellow) [reply components/image/__tests__/index.test.tsx:97]: 不能，本地跑了下
image
只有点击image才会渲染 `.ant-image-preview-root`的dom
[可以看这个demo](https://ant.design/components/image-cn/#components-image-demo-basic)，点击image才会生成dom，才能监听onload

Comment (afc163) [reply components/image/__tests__/index.test.tsx:97]: 我意思是去掉 `forceRender: true` 这句

Comment (binyellow) [reply components/image/__tests__/index.test.tsx:97]: 哦哦哈哈 **不能**
image

**Review by afc163 — APPROVED** [review]

**Pull request merged**